### ISSUE TITLE
Interactive element does not meet minimum size nor spacing - Siteimprove - CKEditor

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#9504]
From #9462

Interactive element does not meet minimum size nor spacing - 2.5.8 Target Size (Minimum) WCAG 2.1 AA

Generally there are buttons on the WYSIWYG toolbar (lists, etc.) that have drop buttons that do not provide the necessary spacing/size. Additional the about text format link below the editor is too small.
